include/joker.h

```diff
@@ -61,6 +61,18 @@ enum JokerEvent
     JOKER_EVENT_ON_BLIND_SELECTED,  // Triggers when selecting a blind (e.g. Dagger, Riff Raff, Madness..)
 };
 
+// These are flags that can be combined into a single u8 and returned by
+// JokerEffect functions to indicate what effects they had
```

Comment:
```suggestion
// JokerEffect functions to indicate which fields of the output JokerEffect are valid
```

Maybe make the documentation a little more specific to what the flags affect.